resources/clients/react/elwing/README.md

```diff
@@ -0,0 +1,83 @@
+# Developer guide
+
+Elwing is a bootstrapped companion app to the Cloudscape design system. Its opinionated
+layout accelerates development time by exposing a simple plugin API allowing a developer
+to focus on defining pages and content rather than app structure.
+
+## Prerequisites
+
+1. Install NodeJS 18
+
+## Run
+
+1. Run `npm i` from the same directory as this readme.
+1. Run `npm start`.
+
+### What is a plugin?
+
+A plugin is just a JavaScript Object containing the following properties:
+
+- `navigationItem` - A Cloudscape `SideNavigationProps` object.
+- `component` - The component that will display when the link in the sidebar is clicked.
+
+## Add a plugin
+
+### ⚠️ Important
+
+The automated steps will overwrite the `manifest.ts` file with the latest
+plugins from the `plugins` directory.
+
+### Add a plugin automatically
+
+A convenience script has been added to support the quick creation of a plugin.
+
+1. Run `npm run create-plugin <plugin-name>`.
+1. Find your plugin in the `plugins` directory.
+1. Modify the generated code to match your use case.
+
+### Add a plugin manually
+
+If you want to create a plugin manually instead of using the script, you can do so.
+
+1. Create a new folder in the plugins directory. For example, `my-plugin`.
+2. Add an `index.ts` file to the folder and populate it using the below as an example.
```

Comment:
2. Add an `index.ts` file to the folder. Refer to the following example to populate the file.